deepchem/utils/rdkit_util.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+import os
+import numpy as np
+import tempfile
+import shutil
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from pdbfixer import PDBFixer
+from simtk.openmm.app import PDBFile
+
+
+class MoleculeLoadException(Exception):
+
+  def __init__(self, *args, **kwargs):
+    Exception.__init__(*args, **kwargs)
+
+
+def get_xyz_from_mol(mol):
+  """
+  returns an m x 3 np array of 3d coords
+  of given rdkit molecule
+  """
+  xyz = np.zeros((mol.GetNumAtoms(), 3))
+  conf = mol.GetConformer()
+  for i in range(conf.GetNumAtoms()):
+    position = conf.GetAtomPosition(i)
+    xyz[i, 0] = position.x
+    xyz[i, 1] = position.y
+    xyz[i, 2] = position.z
+  return (xyz)
+
+
+def add_hydrogens_to_mol(mol):
+  molecule_file = None
+  try:
+    molecule_file = str(tempfile.NamedTemporaryFile().name)
+    Chem.MolToPDBFile(mol, molecule_file)
+    fixer = PDBFixer(filename=molecule_file)
+    fixer.addMissingHydrogens(7.4)
```

Comment:
does this add Hs to the ligand as well? IIRC this is probably using something like a pKa table lookup for amino acids (and probably doesn't do anything for the ligand Hs)